pkg/fastforward/fastforward.go

```diff
@@ -82,6 +85,11 @@ func (f *FastForward) Run() (err error) {
 		options.FastForward = true
 		options.NoMock = f.options.NoMock
 		options.Stream = true
+
+		if f.options.GCPProjectID != "" {
+			options.Project = f.options.GCPProjectID
+		}
```

Comment:
I would say we always set this:
```suggestion
		options.Project = f.options.GCPProjectID
```